backend/database_handler/validators_registry.py

```
@@ -32,6 +34,10 @@ def __init__(self, session: Session):
 
     def _get_validator_or_fail(self, validator_address: str) -> Validators:
         """Private method to check if an account exists, and raise an error if not."""
+        try:
+            validator_address = to_checksum_address(validator_address)
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)